_inc/lib/widgets.php

```diff
@@ -715,14 +714,13 @@ static function insert_widget_in_sidebar( $widget_id, $widget_options, $sidebar
 	}
 
 	/**
-	 * Update the content of an exisitng widget in a given sidebar.
+	 * Update the content of an exisiting widget in a given sidebar.
```

Comment:
Should be `existing` instead of `exisiting`

Reply:
🤦‍♀️

Reply:
s-e-r-i-o-u-s-l-y